pkgs/tools/package-management/nix-upfetch/default.nix

```diff
@@ -0,0 +1,102 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, rustPlatform, pkgconfig, ncurses
+, asciidoc, docbook_xml_dtd_45, docbook_xsl, libxml2, libxslt
+, libredirect, coreutils, gawk, gnugrep, gnused, jq, nix, nix-prefetch }:
+
+with stdenv.lib;
+
+let
+  nix-update-fetch = rustPlatform.buildRustPackage rec {
+    name = "${pname}-${version}";
+    pname = "nix-update-fetch";
+    version = "0.1.0";
+
+    src = fetchFromGitHub {
+      owner = "msteen";
+      repo = "nix-update-fetch";
+      rev = "bc201ec28be669f67501058e97b93e6a6e7b4e07";
+      sha256 = "09qqk9wcbbnwv0rkmr18k4pvaf6xpbrfci7ki44v8m6p4pwvqczw";
+    };
+
+    RUSTC_BOOTSTRAP = 1;
+
+    buildInputs = [ pkgconfig ncurses ];
+
+    cargoSha256 = "0g2gmmhx2gcb02yqmzavx7fqyvdblgg16rhq10rw2slnrmsz84k6";
+
+    meta = {
+      description = "Update a call made to a fetcher function call and its surrounding bindings";
+      homepage = https://github.com/msteen/nix-update-fetch;
+      license = licenses.mit;
+      maintainers = with maintainers; [ msteen ];
+      platforms = platforms.all;
+    };
+  };
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

Comment:
This is not necessary – mkDerivation does that by default.

Reply:
I had it for backwards compatibility, since I developed / tested it under 18.09, but since this PR is based on master, I can indeed remove it.

Reply:
I also tried removing it at `rustPlatform.buildRustPackage`, but it still expects a name attribute, so I will keep it there.